packages/bitcore-client/bin/wallet-send

```diff
@@ -40,14 +43,21 @@ const main = async () => {
     let feeRate = feeRateParam;
     if (!feeRate) {
       const feerate = await wallet.getNetworkFee();
-      feeRate = JSON.parse(feerate).feerate;
+      feeRate = feerate.feerate;
     }
     let nonce;
-    if (!['BTC', 'BCH', 'DOGE', 'LTC'].includes(chain)) {
+    let feeUnit;
+    if (wallet.chain.toUpperCase() === 'XRP') {
+      feeRate = feeRateParam == null ? feeRate : xrpl.xrpToDrops(feeRateParam); // convert feeRateParam from XRP to drops
+      console.log('Fee rate:', feeRate, 'drops');
+    } else if (!wallet.isUtxoChain()) {
       nonce = await wallet.getNonce();
-      feeRate = Math.ceil(feeRateParam == null ? feeRate : (feeRateParam * 1e9)); // convert feeRateParam from Gwei to Wei
+      feeRate = feeRateParam == null ? feeRate : Web3.utils.toWei(feeRateParam.toString(), 'gwei').toNumber(); // convert feeRateParam from Gwei to Wei
```

Comment:
Getting an error here about `toNumber` not existing. You can solve by wrapping the logic in `Number(...)`.